cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -277,76 +271,32 @@ macro(configure_sdk_unix name architectures)
 
   foreach(arch ${architectures})
     if("${prefix}" STREQUAL "ANDROID")
-      if(NOT "${SWIFT_ANDROID_NDK_PATH}" STREQUAL "")
-        set(SWIFT_SDK_ANDROID_ARCH_${arch}_LIBC_INCLUDE_DIRECTORY "${SWIFT_ANDROID_NDK_PATH}/sysroot/usr/include" CACHE STRING "Path to C library headers")
-        set(SWIFT_SDK_ANDROID_ARCH_${arch}_LIBC_ARCHITECTURE_INCLUDE_DIRECTORY "${SWIFT_ANDROID_NDK_PATH}/sysroot/usr/include" CACHE STRING "Path to C library architecture headers")
-      elseif(NOT "${SWIFT_ANDROID_NATIVE_SYSROOT}" STREQUAL "")
-        set(SWIFT_SDK_ANDROID_ARCH_${arch}_LIBC_INCLUDE_DIRECTORY "${SWIFT_ANDROID_NATIVE_SYSROOT}/usr/include" CACHE STRING "Path to C library headers")
-        set(SWIFT_SDK_ANDROID_ARCH_${arch}_LIBC_ARCHITECTURE_INCLUDE_DIRECTORY "${SWIFT_ANDROID_NATIVE_SYSROOT}/usr/include" CACHE STRING "Path to C library architecture headers")
-      else()
-        message(SEND_ERROR "Couldn't find LIBC_INCLUDE_DIRECTORY for Android")
-      endif()
+      swift_android_sysroot(android_sysroot)
+      set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${android_sysroot}")
```

Comment:
Using `_SYSROOT` instead of `_PATH` would be less confusing IMO.

Reply:
Maybe, but this is the variable name chosen by Swift for the sysroot for all OS/ARCH combos, [as can be seen earlier in this file](https://github.com/apple/swift/blob/00117e875081c1d9bbf4591eda5853028edc98fd/cmake/modules/SwiftConfigureSDK.cmake#L59) and used elsewhere in the same generic way. If you'd like, maybe that could be changed in a later pull for all OS's.